gallery/src/components/NxTextInput/NxTextInputPage.tsx

```diff
@@ -395,7 +395,9 @@ const NxTextInputPage = () =>
       Examples of <NxCode>NxTextInput</NxCode>s using
       the <NxCode>long</NxCode> modifier, which makes them wider. Additionally, the second input demonstrates that
       the <NxCode>id</NxCode>, <NxCode>aria-required</NxCode>, and <NxCode>aria-describedby</NxCode> props are
-      passed down directly to the <NxCode>&lt;input&gt;</NxCode>/<NxCode>&lt;textarea&gt;</NxCode>.
+      passed down directly to the <NxCode>&lt;input&gt;</NxCode>/<NxCode>&lt;textarea&gt;</NxCode>. That input
+      is additional wrapped within a label – this is done to appease accessibility standards checking tools which
```

Comment:
```suggestion
      is additionally wrapped within a label – this is done to appease accessibility standards checking tools which
```

Reply:
This inclusion in the PR confused me for a second until I realized it's being merged into RSC-1550 and not main!